oc-chef-pedant/spec/api/status_spec.rb

```diff
@@ -1,12 +1,12 @@
 
-describe "Testing Status API with config", :config do
-  let (:config) { JSON.parse(IO.read("/etc/opscode/chef-server-running.json"))['private_chef'] }
+describe "Testing Status API" do
   context "Status API", :status, :smoke do
     it "GET /_status should respond with valid health data" do
       r = get("#{platform.server}/_status", superuser)
       r.should look_like(status_for_json: 200)
       data = JSON.parse(r)
-      data.has_key?("server_version").to_s.should eql config['opscode-erchef']['include_version_in_status'].to_s
+      include_version_in_status = Pedant::Config.include_version_in_status
+      data.has_key?("server_version").to_s.should eql include_version_in_status.to_s
```

Comment:
I know we were previously doing this, but should we be forcing to string? The config option is a boolean value.

Reply:
Ya I think forcing to string is not needed. I will try this out and see.